documentation/working-with-testcafe-studio/debugging-tests.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+title: Debugging Tests
+permalink: /documentation/working-with-testcafe-studio/debugging-tests
+---
+# Debugging Tests
+
+This topic provides information about debugging tests and contains the following sections.
+
+* [Debugging Recorded Tests](#debugging-recorded-tests)
+  * [Features Useful for Debugging](#features-useful-for-debugging)
+* [Debugging Coded Tests](#debugging-coded-tests)
+
+## Debugging Recorded Tests
+
+To help you understand what is happening in your web page during test run, TestCafe Studio provides a debug mode. In this mode, TestCafe Studio pauses running a test and allows you to debug the tested page using the browser's developer tools: check the web page state, the location of DOM elements, their CSS styles, and so on.
+
+You can activate the debug mode in one of the following ways:
+
+* Add the **Debug** action to a particular place in a test. When test execution reaches this action, it pauses.
+* Enable the **Debug mode** option for a run configuration in the **Run Configurations** dialog. In this case, test run is paused before the first test action.
+* Enable the **Pause the test when it fails** option for a run configuration in the **Run Configurations** dialog. TestCafe Studio enters the debug mode when a test fails.
+
+IMAGE
+
+Once the test run is paused, you can open browser's developer tools and debug.
+
+The browser window's footer displays buttons that allow you to ...
```

Comment:
> The browser window's footer displays buttons that allow you to ...

Wow, that's the use of ellipsis. Moreover, the list items do not continue this sentence,

```
The browser window's footer displays buttons that allow you to

* unlock the tested page and interact with its elements - the **Unlock Page** button,
* continue test execution - the **Resume** button,
* go to the next test action - the **Next Action** button.
```

Reply:
or we can use parentheses

```
The browser window's footer displays buttons that allow you to

* unlock the tested page and interact with its elements (**Unlock Page** button),
* continue test execution (**Resume** button),
* go to the next test action (**Next Action** button).
```